mne/time_frequency/spectrum.py

```diff
@@ -1229,21 +1303,34 @@ def __init__(
         data,
         info,
         freqs,
+        method="unknown",
+        mt_weights=None,
```

Comment:
**[Cont. from above]** `method` is still by default `"unknown"`, but in cases where the dims of `data` are ambiguous, this can be specified to a particular spectral method.

There is also an `mt_weights` param required for complex multitaper data which the user would need to store and which is required for plotting any complex data (when converting complex -> power).